example/lib/support/inner_shadow.dart

```diff
@@ -25,11 +25,12 @@ class InnerShadow extends SingleChildRenderObjectWidget {
 
   @override
   RenderInnerShadow createRenderObject(BuildContext context) {
-    return RenderInnerShadow()
-      ..color = color
```

Comment:
Hmm, wonder why I did it like this. Possibly a copy and paste job without much thought.

Reply:
Oh, no, I remember why. This is the pattern followed by render objects in the framework, because render objects are longer lived and are updated over time as their widgets/context changes.

They avoid the `late` by having field defaults.

But all good. This change is perfect. Thank you. :)